validator/webui/index.html

```diff
@@ -299,18 +301,35 @@
         background-size: 40px 40px;
         background-position: center left;
       }
-      .ampproject a:link { color: #0379C4; text-decoration: none; }
-      .ampproject a:visited { color: #0379C4; text-decoration: none; }
-      .ampproject a:hover { color: #0379C4; text-decoration: none; }
-      .ampproject a:active { color: #0379C4; text-decoration: none; }
+      .ampproject-right {
+        color: #0379C4;
+        margin: 10px 0;
+      }
+      .ampproject-link a:link { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:visited { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:hover { color: #0379C4; text-decoration: none; }
+      .ampproject-link a:active { color: #0379C4; text-decoration: none; }
+
+      .flex-horizontal {
+        @apply(--layout-horizontal);
+        @apply(--layout-center);
+        margin-bottom: 10px;
+      }
+      .flexchild {
+      }
     </style>
     <template>
-      <paper-toolbar>
-        <div class="ampproject">
+      <paper-toolbar "container flex-horizontal">
```

Comment:
Good catch. Interestingly, if I put the class= it stops working and if I remove the entire container flex horizontal thing it keeps working. So I must be getting the correct behavior by default for some reason, will go with that if it's cool. :-)